ui/app/pages/send/send-content/send-gas-row/send-gas-row.component.js

```diff
@@ -89,7 +121,17 @@ export default class SendGasRow extends Component {
       convertedCurrency={convertedCurrency}
       gasLoadingError={gasLoadingError}
       gasTotal={gasTotal}
-      onReset={resetGasButtons}
+      onReset={() => {
+        return new Promise((resolve, reject) => {
```

Comment:
The function passed to the `Promise` constructor is executed synchronously and we're not doing anything async in the function so we can leave it out. We should be able to do something like:

```js
onReset={() => {
  resetGasButtons()
  if (maxModeOn) {
    this.setMaxAmount()
  }
}}
```

Reply:
Awesome, there were other parts in which I used a promise that were not necessary, thank you